### ISSUE TITLE
11-font.js crashes Sketch

### ISSUE THREAD

**Issue by ff6347** [automat/sketch-context2d#3]
Hi. I've been running some of the tests and ran into the first issue at test/CanvasRenderingContext2d-API/11-font.js it crashes my Sketch (updated to v 39.1)

**Comment by ff6347** [comment]
```
/Users/icke/Desktop/sketch-context2d/index.js:61
            throw new Error(err || stderr);
            ^

Error: 2016-07-28 08:36:24.394 coscript[18608:2519833] Exception: (null)

    at /Users/icke/Desktop/sketch-context2d/index.js:61:19
    at ChildProcess.exithandler (child_process.js:193:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:492:12)
```

**Comment by automat** [comment]
I see, works fine with 3.8.x, crashes with 3.9.x. Need to update to 3.9. Putting error log here for reference: 

```
Application Specific Information:
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[MSLayerGroup addLayerOfType:]: unrecognized selector sent to instance 0x7f9118f9ad70'
```

**Comment by automat** [comment]
Fixed with f76dbc4 , updated text layer creation to `Sketch 3.9`.
